apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -388,138 +250,65 @@ class InstructionsCSF extends React.Component {
-      scrollTo(contentContainer, lastChild.offsetTop - 10);
-    } else {
-      scrollBy(contentContainer, contentContainer.scrollHeight);
-    }
-  }
-
-  /**
-   * Handle a click to the hint display bubble (lightbulb)
-   */
-  handleClickBubble = () => {
-    // If we don't have authored hints to display, clicking bubble shouldnt do anything
-    if (this.props.hasAuthoredHints && this.props.hasUnseenHint) {
-      this.setState({
-        promptForHint: true
-      });
-      if (this.props.collapsed) {
-        this.props.handleClickCollapser();
-      }
-    }
-  };
-
   dismissHintPrompt = () => {
     this.setState({
       promptForHint: false
     });
   };
 
-  showHint = () => {
-    this.dismissHintPrompt();
-    this.props.showNextHint();
-    this.props.clearFeedback();
-  };
-
-  shouldDisplayHintPrompt() {
+  shouldDisplayHintPrompt = () => {
     return this.state && this.state.promptForHint && !this.props.collapsed;
-  }
-
-  shouldDisplayCollapserButton() {
-    // if we have "extra" (non-instruction) content, we should always
-    // give the option of collapsing it
-    if (
-      this.props.hints.length ||
-      this.shouldDisplayHintPrompt() ||
-      this.props.feedback
-    ) {
-      return true;
-    }
-
-    // Otherwise, only show the button if we have two versions of
-    // instruction we want to toggle between
-    return this.props.longInstructions && !this.shouldIgnoreShortInstructions();
-  }
+  };
 
-  shouldIgnoreShortInstructions() {
-    // If we have no short instructions, always ignore them.
-    if (!this.props.shortInstructions) {
-      return true;
-    }
+  hasShortInstructions = () => {
+    return (
+      !!this.props.shortInstructions &&
+      !this.shortAndLongInstructionsAreEquivalent()
+    );
+  };
 
-    // Otherwise, if we have no long instructions, never ignore the short ones.
-    // Note that we would only decide to ignore short instructions in
-    // the absense of long instructions when the short instructions
-    // themselves were less than 10 characters, which can easily happen
-    // in ideo- or logographic languages.
-    if (!this.props.longInstructions) {
+  shortAndLongInstructionsAreEquivalent() {
+    if (!this.props.shortInstructions || !this.props.longInstructions) {
       return false;
     }
-
-    // if short instructions and long instructions have a Levenshtein
-    // Edit Distance of less than or equal to 10, ignore short
-    // instructions and only show long.
-    let dist = levenshtein(
+    const dist = levenshtein(
       this.props.longInstructions,
       this.props.shortInstructions
     );
     return dist <= 10;
   }
 
-  shouldDisplayShortInstructions() {
-    return (
-      !this.shouldIgnoreShortInstructions() &&
-      (this.props.collapsed || !this.props.longInstructions)
-    );
-  }
+  requestHint = () => {
+    this.setState({
+      promptForHint: true
+    });
+    this.props.collapsed && this.props.handleClickCollapser();
```

Comment:
Is the intent here to ensure that we are not collapsed?

Reply:
Yeah I believe so, hints are only displayed in the expanded state